media/css/firefox/ios.less

```diff
@@ -113,28 +113,28 @@ a.go {
         }
     }
 
-    h2 {
-        .font-size(40px);
-        color: #bee1f5;
-        line-height: 1.2;
-        margin: 0 0 1em;
-
-        @media only screen and (max-width: @breakDesktop) {
-            .font-size(32px);
-        }
-
-        @media only screen and (max-width: @breakTablet) {
-            .font-size(22px);
-            text-align: center;
-        }
-    }
-
     a:link,
     a:visited {
         color: #fff;
     }
 
     .get-fxios {
+        &> h2 {
```

Comment:
Smallest nit in the world (only mentioning because you'll probably update the JS comment) - no space between `&` and `>`.